### ISSUE TITLE
about the "adapt" in maml.py

### ISSUE THREAD

**Issue by liujianzhao6328057** [tristandeleu/pytorch-maml#5]
Hi,  thanks for your implementation. I have read your code but I have some problems. Why the params is set to "None" in the "adapt" in maml.py ? Why not the current params value?

    def adapt(self, inputs, targets, is_classification_task=None,
              num_adaptation_steps=1, step_size=0.1, first_order=False):
        if is_classification_task is None:
            is_classification_task = (not targets.dtype.is_floating_point)
        **params = None**   #_____________________________________________>I cannot understand

        results = {'inner_losses': np.zeros(
            (num_adaptation_steps,), dtype=np.float32)}

        for step in range(num_adaptation_steps):
            **logits = self.model(inputs, params=params)** #----------------------->params=None?
            inner_loss = self.loss_function(logits, targets)
            results['inner_losses'][step] = inner_loss.item()

            if (step == 0) and is_classification_task:
                results['accuracy_before'] = compute_accuracy(logits, targets)

            self.model.zero_grad()
            params = update_parameters(self.model, inner_loss,
                step_size=step_size, params=params,
                first_order=(not self.model.training) or first_order)

        return params, results

**Comment by tristandeleu** [comment]
Hi, sorry for the late reply. The `params=None` corresponds to the standard behavior of a PyTorch module: the module uses the parameters stored in the `Module` object. Here, this means that it uses the initialization (because the initialization / the meta-parameters are stored in the `MetaModule`)
